### ISSUE TITLE
Can't install project

### ISSUE THREAD

**Issue by reorg-hle** [jolicode/slack-php-api#6]
I'm just following the instructions in the readme

```
composer require php-http/curl-client guzzlehttp/psr7
composer require jolicode/slack-php-api
```
```
 [InvalidArgumentException]
  Could not find a version of package jolicode/slack-php-api matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
```

Php version is 

```$ php -v
PHP 7.1.1-1+deb.sury.org~trusty+1 (cli) (built: Jan 20 2017 09:43:29) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2017 Zend Technologies
    with Zend OPcache v7.1.1-1+deb.sury.org~trusty+1, Copyright (c) 1999-2017, by Zend Technologies
    with Xdebug v2.6.1, Copyright (c) 2002-2018, by Derick Rethans
```

**Comment by pyrech** [comment]
I just released v1.0 so you should be able to install it right now 😉

**Comment by reorg-hle** [comment]
All good, thanks!
